docs/get-started/tooling/cross-chain/layerzero.mdx

```diff
@@ -1,61 +1,110 @@
 ---
 title: LayerZero Omnichain Messaging
 image: /img/socialCards/layerzero-omnichain-messaging.jpg
+description: "Omnichain interoperability protocol enabling secure cross-chain communication for Linea applications. Connect to 100+ blockchains, send messages, and deploy omnichain tokens."
 ---
 
-LayerZero is an omnichain interoperability protocol that enables cross-chain 
-messaging. Applications built on Linea can use the LayerZero protocol to 
-seamlessly connect to 35+ supported blockchains.
+[LayerZero](https://layerzero.network) is an **omnichain interoperability protocol** that enables smart contracts to seamlessly communicate between different blockchain networks. With LayerZero V2, applications deployed on Linea can connect and interact with 100+ supported blockchains through secure, configurable messaging channels.
 
-LayerZero Endpoint has been deployed on Linea for builders to leverage omnichain 
-messaging in their dapps. LayerZero's Endpoint ULNv2 validation library relies 
-on two parties, the Oracle and Relayer, to transfer messages between onchain 
-smart contracts known as endpoints. When LayerZero sends a message from Linea to 
-chain `X`, the message is routed through the endpoint on Linea to the ULNv2 
-validation library. The ULNv2 library notifies the Oracle and Relayer of the 
-message and its destination chain `X`. The Oracle forwards the packet hash to 
-the endpoint on chain `X`, and the Relayer submits the packet to be verified 
-onchain against the hash, and delivers the message.
+## Key Features
 
-As a developer, you can leverage LayerZero's Endpoint contract by using our 
-`LzApp` interface for your User Application (UA). The Endpoint allows UAs to 
-configure the Messaging Library for sending and receiving verified messages and 
-guarantees the message-delivering ordering across all messaging libraries:
+LayerZero enables powerful cross-chain capabilities for builders on Linea:
 
-- `Send()`: the message will be sent through the endpoint first, and then 
-redirected to the UA-configured Messaging Library. 
-- `Receive()`: the message will be verified at the Messaging Library first, then 
-forwarded to the endpoint and eventually delivered to the UA.
+1. **Cross-chain Messaging** - Send arbitrary messages and data between contracts on different chains
+2. **Omnichain Tokens** - Deploy tokens (fungible `OFT` and non-fungible `ONFT`) that work seamlessly across multiple chains
+3. **External Chain Data Access** (`lzRead`) - Fetch and compute on-chain state from other networks
+4. **Composed Messages** - Chain multiple cross-chain operations together
 
-Learn how to integrate LayerZero into your contracts [here](https://docs.layerzero.network/v1).
+## How it works
+
+1. **DVNs** independently verify that a message is valid, waiting for a configured number of block confirmations on the source chain.
```

Comment:
I think we need to spell out DVN in full here. After that, using the initialism is fine